proxy/src/proxy.rs

```diff
@@ -85,6 +86,9 @@ impl ProxyConnection {
             }
         };
 
+        // TODO send BackendKeyData with CancelToken.
+        //      Should I add a variant to BeMessage?
```

Comment:
Yeah.

Postgres responds with BackendKeyData right after auth is done: https://github.com/zenithdb/postgres/blob/main/src/backend/tcop/postgres.c#L4160

I could think of two ways to implement that:

1) proxy generates its own (pid, cancelKey) pair and send it to the client; that pair should be mapped to the 
corresponding pair from postgres that you can somehow take out of rust-postgres connection

2) take (pid, cancelKey) out of rust-postgres connection and send it to the client before sending ReadyForQuery

Second is preferable I think but it potentially changes order of messages (or now -- not sure) so may not work with the client.